src/components/map/CountriesTileLayer.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { useMap } from "react-leaflet";
+import L from 'leaflet';
+import { layerStyle, highlightStyles } from './MapStyle';
+import _ from "lodash";
+import { createAltPopup } from "../../utils/mapUtils"
+import ReactDOMServer from 'react-dom/server';
+// eslint-disable-next-line no-unused-vars
+import VectorGrid from "leaflet.vectorgrid";
+
+const toggleCountrySelection = (e, layer, highlightStyle) => {
+  // only change selection style on countries that have seroprev data
+  const layerStyle = layer._overriddenStyles[e.layer.properties.CODE]; //TODO: refactor this kinda ugly
+
+  if (layerStyle) 
+    layer.setFeatureStyle(e.layer.properties.CODE, highlightStyle);
+}
+
+export default function CountriesTileLayer(props) {
+
+  const { url, records, zIndex } = props;
```

Comment:
Can we either rename records to `mapRecords` or make a comment here explicitly reminding developers that the "records" here refer to a prop rather than the "record" data model that we use in the context (and have become very accustomed to)

Reply:
renamed to mapRecords